tests/system/robot/chromeTests.py

```diff
@@ -140,6 +140,138 @@ def test_mark_aria_details_FreeReviewCursor():
 	exercise_mark_aria_details()
 
 
+def test_mark_aria_details_role():
+	_chrome.prepareChrome(
+		"""
+		<div class="editor" contenteditable spellcheck="false" role="textbox" aria-multiline="true">
+			<p>
+				<span aria-details="endnote-details">doc-endnote,</span>
+				<span aria-details="footnote-details">doc-footnote,</span>
+				<span aria-details="comment-details">comment,</span>
+				<span aria-details="definition-details-as-role">definition,</span>
+				<span aria-details="definition-details-as-tag">definition,</span>
+				<span aria-details="unknown-details">form</span>
+			</p>
+		</div>
+		<div>
+			<p>
+				<!-- Supported by Chrome attribute details-roles -->
+				<div id="endnote-details" role="doc-endnote">details with role doc-endnote</div>
+				<div id="footnote-details" role="doc-footnote">details with role doc-footnote</div>
+				<div id="comment-details" role="comment">details with role comment</div>
+				<div id="definition-details-as-role" role="definition">details with role definition</div>

```

Comment:
I think this needs a `role="term"` next to it for it to be valid.

Reply:
I still don't think this is quite right. [Shouldn't it be](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/definition_role):
```suggestion
				<p><span role="term">definition</span>:	<span id="definition-details-as-role" role="definition">details with role definition</span>
				</p>
```

Reply:
Fixed by 55eeb9f